blazor-toc.html

```diff
@@ -1711,9 +1711,15 @@
 			</li>
 			<li>
 				<a href="/blazor/datagrid/pdf-export">Export to PDF</a>
+				<ul>
+					<li><a href="/blazor/datagrid/template-pdf-export">Template Exporting</a></li>
+				</ul>
 			</li>
 			<li>
 				<a href="/blazor/datagrid/excel-exporting">Export to Excel</a>
+				<ul>
+					<li><a href="/blazor/datagrid/template-excel-export">Template Exporting</a></li>
```

Comment:
Title Will be excel exporting with templates like as EJ2 documentation

https://ej2.syncfusion.com/documentation/grid/excel-export/exporting-with-templates